localgov_base.theme

```diff
@@ -87,12 +87,29 @@ function localgov_base_form_system_theme_settings_alter(&$form, FormStateInterfa
     '#default_value' => theme_get_setting('localgov_base_localgov_guides_vertical_navigation'),
     '#description'   => t('This will display the guide navigation vertically above the guide.'),
   ];
+
+  $form['localgov_base_mobile_breakpoint_js'] = [
+    '#type'          => 'textfield',
+    '#title'         => t('Mobile breakpoint for JavaScript'),
+    '#default_value' => theme_get_setting('localgov_base_mobile_breakpoint_js'),
+    '#description'   => t('This is the mobile breakpoint in pixels. This is used by the JS to determine when to show the mobile menu. If left blank, this will default to 768px.'),
+    '#attributes' => [
+      'placeholder' => '768',
+    ],
+
```

Comment:
I'd be tempted to add a validation function here, like 

```php
$form['#validate'][] = 'localgov_base_theme_settings_validate';
```

In concert with making the setting a number field, this could eliminate the need to handle the determination of the value in javascript since the value would be _guaranteed_ numeric (e.g. the validation could test `is_numeric()` to pass or fail validation.

Reply:
Validation added.